recipes/mcxtrace-suite/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = "3.4.9" %}
+{% set sha_value = "ccab20cca6e842d5b643ae30c0cd3330c3af31375b205f251cfafc086fd8000b" %}
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: mcxtrace-suite
+  version: {{ version }}
+
+source:
+  url: https://github.com/McStasMcXtrace/McCode/archive/mcxtrace-{{ version }}.tar.gz
+  sha256: {{ sha_value }}
+  folder: src
+
+build:
+  number: 2
+
+outputs:
+
+  - name: mcxtrace-core
+    script: install-files-core.sh  # [not win]
+    script: install-files-core.bat  # [win]
+    build:  # [win]
+      skip: true  # [win]
+    requirements:
+      build:
+        - cmake
+        - make  # [not win]
+        - {{ compiler('c') }}
+        - flex  # [not win]
+        - bison  # [not win]
+        - winflexbison  # [win]
+        - mslex  # [win]
+      host:
+        - dirent  # [win]
+      run:
+        - python
+        - {{ compiler('c') }}
+        - bash  # [unix]
+        - pyaml # for mxrun
+        - ply # for traces from mxrun
```

Comment:
Can you remove these comments? They don't seem to add value/info to the recipe. If you really need them, move them to the line above to make them more readable and avoid confusion with the pre-processor selectors used in the recipe.